sensor/cli/main.py

```diff
@@ -0,0 +1,16 @@
+import click
+import commands
+
+
+@click.group()
+def cli() -> None:
+    pass
+
+
+cli.add_command(commands.start, name="start")
+cli.add_command(commands.stop, name="stop")
+cli.add_command(commands.is_running, name="is-running")
+
+
+if __name__ == "__main__":
+    cli.main(prog_name="insert-name-here-cli")
```

Comment:
Can you explain the goal of the CLI a bit more to me, what it's used for, so I can understand it better?

One thing I'm unsure of is how stable the connection would need to be for SSH to work ok. I'm thinking about sensors in mountainous areas and such.

Reply:
Sure! The CLI is very helpful on our development station (has Ethernet). Additionally, it can be used to _"start automation as a background process if it is not already running"_ to simply run the start command periodically (if anything crashes completely) or on system startup.

I could build that logic into the `run.py` script itself, but then I wouldn't have a quick and clean way to stop it. I wrote it for https://github.com/tum-esm/automated-proffast-pylot so I could simply reuse it.

Reply:
> [...] to simply run the start command periodically (if anything crashes completely) or on system startup.

That makes sense to me 👍 

You could think about using [`screen` instead of `nohup`](https://stackoverflow.com/a/16809370). May be more reliable than searching for PIDs, and additionally, you could get back into the session, to see log output and such.